lib/src/BatchResponseContent.d.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * @module BatchResponseContent
+ */
+/**
+ * @interface
+ * Signature represents key value pair object
+ */
+interface KeyValuePairObject {
+    [key: string]: any;
+}
+/**
+ * @interface
+ * Signature representing Batch response payload
+ * @property {KeyValuePairObject[]} responses - An array of key value pair representing response object for every request
+ * @property {string} @nextLink - The nextLink value to get next set of responses in case of asynchronous batch requests
+ */
+interface BatchResponsePayload {
```

Comment:
We should see if we can thing of a different name than payload.  Having one class called Content and another called Payload is a bit confusing.  Now seeing your code I have a better understanding of why you tried to name it this way.